dashboard/test/models/concerns/user/terms_of_service_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+class TermsOfServiceTest < ActiveSupport::TestCase
+  include Minitest::RSpecMocks
+
+  let(:teacher) {build(:teacher, :with_terms_of_service)}
```

Comment:
nit, we prefer to `omit_parentheses` when making factory bot calls: 
```suggestion
  let(:teacher) {build :teacher, :with_terms_of_service}
```
I will be enforcing this rule in a PR soon.